packages/extension-api/src/extension-api.d.ts

```diff
@@ -277,10 +277,17 @@ declare module '@podman-desktop/api' {
   }
   export type CheckResultLink = Link;
 
+  export interface CheckResultFixCommand {
+    id: string;
+    title: string;
+  }
+
   export interface CheckResult {
     successful: boolean;
     description?: string;
+    docLinksDescription?: string;
     docLinks?: CheckResultLink[];
+    fixFailedCheckCommand?: CheckResultFixCommand;
```

Comment:
IMHO as we are in a CheckResult object we don't need to add failed or check inside the attribute
failed as it depends on the successful response and check as we're inside a CheckResult

so maybe shorter like `fixCommand?`